react-cra/src/pages/WebComponents.tsx

```diff
@@ -64,201 +77,211 @@ export default function WebComponents() {
+        <sdds-link href="#" target="_self">
+          This is a link.
+        </sdds-link>
+      </div>
+
+      <div>
+        <div className="sdds-headline-02 sdds-u-pb1">Message</div>
+        <sdds-message
+          type="information"
+          header="Message header"
+          mode-variant="inherit from parent"
         >
-          <sdds-dropdown-option value="option-1" tabIndex={0} disabled>
-            Option 1
-          </sdds-dropdown-option>
-          <sdds-dropdown-option value="option-2" tabIndex={0}>
-            Option 2
-          </sdds-dropdown-option>
-          <sdds-dropdown-option value="option-3" tabIndex={0}>
-            Option 3
-          </sdds-dropdown-option>
-        </sdds-dropdown>
+          Longer message text can be placed here. Longer message text can be
+          placed here.
+        </sdds-message>
+      </div>
 
-        <div>
-          <div className="sdds-headline-02 sdds-u-pb1">Link</div>
-          <sdds-link href="#" target="_self">
-            This is a link.
-          </sdds-link>
-        </div>
+      <div>
+        <div className="sdds-headline-02 sdds-u-pb1">Radio Button</div>
+        <sdds-radio-button
+          name="rb-example"
+          value="option1"
+          radio-id="option-1"
+          checked
+        >
+          Label text 1
+        </sdds-radio-button>
 
-        <div>
-          <div className="sdds-headline-02 sdds-u-pb1">Message</div>
-          <sdds-message
-              type="information"
-              header="Message header"
-              mode-variant="inherit from parent"
+        <sdds-radio-button
+          name="rb-example"
+          value="option2"
+          radio-id="option-2"
+        >
+          Label text 2
+        </sdds-radio-button>
+      </div>
+      <div>
+        <div className="sdds-headline-02 sdds-u-pb1">Slider</div>
+        <sdds-slider
+          id="sdds-slider"
+          min="0"
+          max="100"
+          value="50"
+          tooltip
+          label="Test label"
+          ticks="4"
+          controls
+          show-tick-numbers
+        ></sdds-slider>
+      </div>
+      <div>
+        <div className="sdds-headline-02 sdds-u-pb1">Tooltip</div>
+        <div className="tooltip-container">
+          <sdds-tooltip
+            placement="bottom"
+            selector="#button-1"
+            text="Text inside tooltip"
+            mouse-over-tooltip="true"
           >
-            Longer message text can be placed here. Longer message text can be
-            placed here.
-          </sdds-message>
-        </div>
-
-        <div>
-          <div className="sdds-headline-02 sdds-u-pb1">Radio Button</div>
-            <sdds-radio-button
-                name="rb-example"
-                value="option1"
-                radio-id="option-1"
-                checked
-            >
-              Label text 1
-            </sdds-radio-button>
+            <p className="sdds-detail-05 sdds-u-m0">
```

Comment:
Please add some spacing here, you will see that the message has a couple of words glued together.